### PR TITLE
Add organization state and go home button

### PR DESCRIPTION
**Pull request by NicoTuxx**
https://github.com/user-attachments/assets/c1c1a60b-5b37-4dbd-8e99-b9a810a06c90

### CODE REVIEW

File: client/src/views/home/HomePage.vue [hunk 1]
```diff
@@ -172,6 +172,14 @@ async function handleQuery(): Promise<void> {
     } else {
       console.error('Could not find the corresponding device');
     }
+  } else if (query.organizationId) {
```
Comment (Max-7) [inline client/src/views/home/HomePage.vue:175]: We should probably rename this to `saasOrganizationId` and check if the device belongs to a Saas organization. It is possible to get multiple devices belonging to an org with the same name on different servers.

File: client/src/views/home/HomePage.vue [hunk 2]
```diff
@@ -172,6 +172,14 @@ async function handleQuery(): Promise<void> {
...
+    const availableDevices = await listAvailableDevices();
+    const device = availableDevices.find((d) => d.organizationId === query.organizationId);
+    if (device) {
+      await onOrganizationSelected(device);
+    } else {
+      console.error('Could not find the corresponding device');
```
Comment (Max-7) [inline client/src/views/home/HomePage.vue:181]: Handle errors.

**Review by Max-7 — CHANGES_REQUESTED** [review]

File: client/src/views/client-area/ClientAreaSidebar.vue [hunk 3]
```diff
@@ -230,11 +230,18 @@ import {
 } from '@ionic/vue';
 import OrganizationSwitchClientPopover from '@/components/organizations/OrganizationSwitchClientPopover.vue';
 import { ClientAreaPages, BillingSystem, isDefaultOrganization } from '@/views/client-area/types';
+import { navigateTo, Routes } from '@/router';
 
 const props = defineProps<{
   organization: BmsOrganization;
   currentPage: ClientAreaPages;
 }>();
+const status = asyncComputed<OrganizationStatusResultData | null>(async () => {
```
[the remark this answers is not in the document]
Comment (NicoTuxx) [reply client/src/views/client-area/ClientAreaSidebar.vue:239]: done

File: client/src/views/client-area/ClientAreaSidebar.vue [hunk 4]
```diff
@@ -263,6 +270,14 @@ async function openOrganizationChoice(event: Event): Promise<void> {
     emits('organizationSelected', data.organization);
   }
 }
+
+async function goToHome(): Promise<void> {
+  if (props.organization) {
```
[the remark this answers is not in the document]
Comment (NicoTuxx) [reply client/src/views/client-area/ClientAreaSidebar.vue:275]: done

File: client/src/views/client-area/ClientAreaSidebar.vue [hunk 5]
```diff
@@ -33,25 +33,25 @@
       </ion-card-header>
 
       <!-- show it only when there is one organization selected -->
-      <template>
+      <template v-if="!isDefaultOrganization(organization) && status">
```
[the remark this answers is not in the document]
Comment (NicoTuxx) [reply client/src/views/client-area/ClientAreaSidebar.vue:36]: yes that's why this check is here, with default org status is null but on line 39 we use status.isFrozen, which is raising an error without the check.

File: client/src/views/home/HomePage.vue (hunk 1 above)
Comment (NicoTuxx) [reply client/src/views/home/HomePage.vue:175]: done

File: client/src/views/home/HomePage.vue (hunk 2 above)
Comment (NicoTuxx) [reply client/src/views/home/HomePage.vue:181]: done

**Review by Max-7 — APPROVED** [review]

**Pull request merged**